compat/media/media_recorder_client.cpp

```diff
@@ -367,3 +383,70 @@ sp<IGraphicBufferProducer> MediaRecorderClient::querySurfaceMediaSource()
     }
     return recorder->querySurfaceMediaSource();
 }
+
+#if ANDROID_VERSION_MAJOR>=8
+status_t MediaRecorderClient::setNextOutputFile(int fd)
+{
+    REPORT_FUNCTION();
+    ALOGV("setNextOutputFile(%d)", fd);
+    Mutex::Autolock lock(recorder_lock);
+    if (recorder == NULL) {
+        ALOGE("recorder is not initialized");
+        return NO_INIT;
+    }
+    return recorder->setNextOutputFile(fd);
+}
+
+status_t MediaRecorderClient::getMetrics(Parcel* reply)
+{
+    REPORT_FUNCTION();
+    ALOGV("MediaRecorderClient::getMetrics");
+    Mutex::Autolock lock(recorder_lock);
+    if (recorder == NULL) {
+        ALOGE("recorder is not initialized");
+        return NO_INIT;
+    }
+    return recorder->getMetrics(reply);
+}
+
+status_t MediaRecorderClient::setInputDevice(audio_port_handle_t deviceId) {
```

Comment:
This and the following functions have { at the end of the function definition line when the functions before this have it in the next line (including all function that were already in the file), please make formatting consistent.